FlorenceBML/BMLacq679/BMLacq679.xml

```diff
@@ -46,66 +46,66 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                            </msItem>
+                            <msItem xml:id="ms_i2.3">
+                                <locus from="105v" to="107r"></locus>
+                                <title ref="LIT2703salota"></title>
+                            </msItem>
+                            <msItem xml:id="ms_i2.4">
+                                <locus from="107r" to="107v"></locus>
+                                <title ref="LIT2257salota"></title>
+                            </msItem>
+                            <msItem xml:id="ms_i2.5">
+                                <locus from="107v" to="108v"></locus>
+                                <title ref="LIT2258salota"></title>
+                            </msItem>
+                            <msItem xml:id="ms_i2.6">
+                                <locus from="108v" to="109r"></locus>
+                                <title ref="LIT2265salota"></title>
+                            </msItem>
+                            <msItem xml:id="ms_i2.7">
+                                <locus from="109r" to="109v"></locus>
+                                <title ref="LIT2274salota"></title>
+                            </msItem>
+                            <msItem xml:id="ms_i2.8">
+                                <locus from="109v" to="110v"></locus>
+                                <title ref="LIT2250salota"></title>
+                            </msItem>
+                            <msItem xml:id="ms_i2.9">
+                                <locus target="#110v"></locus>
+                                <title ref="LIT2271salota"></title>
+                            </msItem>
+                            <msItem xml:id="ms_i2.10">
+                                <locus from="110v" to="111r"></locus>
+                                <title ref="LIT1566hababa"></title>
+                            </msItem>
+                            <msItem xml:id="ms_i2.11">
+                                <locus from="111r" to="112v"></locus>
+                                <title ref="LIT2251salota"></title>
+                            </msItem>
+                            <msItem xml:id="ms_i2.12">
+                                <locus from="112v" to="113r"></locus>
+                                <title ref="LIT2259salota"></title>
+                            </msItem>
+                            <msItem xml:id="ms_i2.13">
+                                <locus from="113r" to="113v"></locus>
+                                <title ref="LIT1827Magnif"></title>
+                            </msItem>
+                            <msItem xml:id="ms_i2.14">
+                                <locus from="113v" to="114r"></locus>
+                                <title ref="LIT2275salota"></title>
+                            </msItem>
+                            <msItem xml:id="ms_i2.15">
+                                <locus target="114r"></locus>
```

Comment:
sorry, # is missing here, otherwise all good

Reply:
Done.